browser/net/url_context.h

```diff
@@ -90,6 +90,8 @@ struct BraveRequestInfo {
   const base::ListValue* referral_headers_list = nullptr;
   BlockedBy blocked_by = kNotBlocked;
   bool cancel_request_explicitly = false;
+  std::string redirect;
```

Comment:
IMO this name is quite confusing: (1) we don't do any redirects with it (2) we already have `new_url` below.
So maybe rename it to `data_url_response` (if this is always a data URL)? Or `stub_response`?

I'd also prefer using `base::Optional` but if it looks wordy/inconvenient I'm ok to leave it as is.

Reply:
also some human-readable description would be useful

Reply:
Yep, the name comes from ABP filter syntax though that we use, and uBlock implements it as a redirect.   I'll update it in most places to `data_url`.